server/utils/emailUtils.js

```diff
@@ -0,0 +1,72 @@
+const nodemailer = require('nodemailer');
+const { emailCreds } = require('../config/index')
+const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: emailCreds,
+      pass: emailCreds
+    }
+  });
+
+exports.sendCourseRegistrationMail = async (data) => {
+    const { user } = data;
+    console.log("User Data is", user);
+    let linkDocuments = `<a>No Documents Available</a>`
+    if(user.documents && user.documents[0]) {
+      console.log("documents are", user.documents[0].cloudStorage)
+      const documents = user.documents[0].cloudStorage;
+      linkDocuments = documents.map(document => `<li><a href="${document.imageUrl}">${document.originalName}</a></li>`).join('');
+    }
+    const subject = "New Registration For Course";
+    const courseMailOptions = {
+        from: emailCreds,
+        to: 'samplforeignmastere@yopmail.com',
+        subject: subject,
+        html:  `<html lang="en">
+    
+        <head>
+            <meta http-equiv="Content-Type" content="text/html charset=UTF-8" />
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+            <meta http-equiv="X-UA-Compatible" content="ie=edge">
+            <link href="https://fonts.googleapis.com/css?family=Quicksand:400,600&display=swap" rel="stylesheet">
+            <title>Foreign Master</title>
+        </head>
```

Comment:
sample mail html is ok for now, later we can add css to make it a beautiful user portfolio mail with all details.

Reply:
Works !